bcf_api_xml/export.py

```diff
@@ -96,86 +123,90 @@ def to_zip(topics, comments, viewpoints):
-    worksheet.write(row, 3, "Auteur",          headerFmt)
-    worksheet.write(row, 4, "Titre",           headerFmt)
-    worksheet.write(row, 5, "Description",     headerFmt)
-    worksheet.write(row, 6, "Date d'échéance", headerFmt)
-    worksheet.write(row, 7, "Statut",          headerFmt)
-    worksheet.write(row, 8, "Priorité",        headerFmt)
-    worksheet.write(row, 9, "Image",           headerFmt)
-    row += 1
-
-    # Create topic rows
-    for topic in topics:
-        topic_guid = topic["guid"]
-        topic_comments = comments.get(topic_guid, [])
-        topic_viewpoints = viewpoints.get(topic_guid, [])
-
-        worksheet.write(row, 0, row, baseFmt)
-        worksheet.write(row, 1, topic.get("index"), baseFmt)
-
-        creation_date = topic.get("creation_date")
-        if creation_date:
-            creation_date = datetime.strptime(creation_date, "%Y-%m-%dT%H:%M:%S.%fZ")
-            worksheet.write_datetime(row, 2, creation_date, dateFmt)
-
-        worksheet.write(row, 3, topic.get("creation_author"), baseFmt)
-        worksheet.write(row, 4, topic.get("title"), baseFmt)
-        worksheet.write(row, 5, topic.get("description"), baseFmt)
-
-        due_date = topic.get("due_date")
-        if due_date:
-            due_date = datetime.strptime(due_date, "%Y-%m-%dT%H:%M:%SZ")
-            worksheet.write_datetime(row, 6, due_date, dateFmt)
-
-        worksheet.write(row, 7, topic.get("topic_status"), baseFmt)
-        worksheet.write(row, 8, topic.get("priority"), baseFmt)
-
-        for viewpoint in topic_viewpoints:
-            if viewpoint.get("snapshot"):
-                snapshot = viewpoint.get("snapshot").get("snapshot_data")
-                img_file = "snapshot-"+viewpoint["guid"]+".png"
-                if ";base64," in snapshot:
-                    _, img_data = snapshot.split(";base64,")
-                    img_data = base64.b64decode(img_data)
-                else:
-                    img_data = requests.get(snapshot).content
-
-                with open(img_file, "wb") as f:
-                    f.write(img_data)
-                with Image.open(img_file) as img:
-                    width, height = img.size
-                scale = 300 / width
-
-                worksheet.set_column_pixels(9, 9, 300)
-                worksheet.set_row_pixels(row, height * scale)
-                worksheet.insert_image(row, 9, img_file, { "x_scale": scale, "y_scale": scale })
-
-                row += 1
-
+    xls_file = io.BytesIO()
+    with xlsxwriter.Workbook(xls_file) as workbook:
+        worksheet = workbook.add_worksheet()
+
+        header_fmt = workbook.add_format({ "align": "center", "bold": True, "bg_color": "#C0C0C0" })
+        base_fmt = workbook.add_format({ "valign": "top" })
+        date_fmt = workbook.add_format({ "valign": "top", "num_format": "yyyy-mm-dd" })
+        comments_fmt = workbook.add_format({ "valign": "top", "text_wrap": True })
+
+        headers = XLS_HEADER_TRANSLATIONS[lang]
```

Comment:
is lang does not exist, fallback to en

Reply:
I already defined `"en"` as [the default value for `lang` here](https://github.com/bimdata/BCF-API-XML-translator/pull/9/files#diff-d04fee44370a0aa1f24bb8bdc75ae6919c6e4d6730fde99542ed73156ba99be7R126).
Is this OK or should I do it another way ?